### PR TITLE
Remove workaround for converting column numbers

### PR DESCRIPTION
**Pull request by yuhan0**
Fixes https://github.com/clojure-emacs/cider/issues/2852

- [x] The commits are consistent with our [contribution guidelines](../blob/master/.github/CONTRIBUTING.md)
- [ ] You've added tests to cover your change(s)
- [x] All tests are passing
- [x] The new code is not generating reflection warnings
- [x] You've updated the README (if adding/changing middleware)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 ### Bugs fixed
 
 * Bump Orchard to 0.5.10, as the 0.5.9 was a bad build. ([Orchard issue](https://github.com/clojure-emacs/orchard/issues/91))
+* Remove workaround for converting column numbers to 0-based indexing. ([Cider issue](https://github.com/clojure-emacs/cider/issues/2852))
```
Comment (bbatsov) [inline CHANGELOG.md:10]: That should go under `master`.

File: src/cider/nrepl/middleware/debug.clj [hunk 2]
```diff
@@ -477,13 +477,12 @@ this map (identified by a key), and will `dissoc` it afterwards."}
                                  ;; Passing clojure.lang.Namespace object
                                  ;; as :original-ns breaks nREPL in bewildering
                                  ;; ways.
+                                 ;; NOTE: column numbers in the response map
```
Comment (bbatsov) [inline src/cider/nrepl/middleware/debug.clj:480]: Probably we should also mention this in the debugger docs.

Comment (yuhan0) [reply src/cider/nrepl/middleware/debug.clj:480]: Do you mean the docs in this repo or Cider's?

Comment (bbatsov) [reply src/cider/nrepl/middleware/debug.clj]: CIDER's. I think we don't have any debugger documentation elsewhere, although we should change this at some point.

**Comment by bbatsov** [comment]
All of the tests are failing now.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,11 +2,16 @@
...
+### Bugs fixed
+
+* Remove workaround for converting column numbers to 0-based indexing. ([Cider issue](https://github.com/clojure-emacs/cider/issues/2852))
+
 ## 0.25.1 (2020-06-04)
 
 ### Bugs fixed
 
 * Bump Orchard to 0.5.10, as the 0.5.9 was a bad build. ([Orchard issue](https://github.com/clojure-emacs/orchard/issues/91))
+* Remove workaround for converting column numbers to 0-based indexing. ([Cider issue](https://github.com/clojure-emacs/cider/issues/2852))
```
Comment (bbatsov) [inline CHANGELOG.md:14]: I think you forgot to delete this.

**Comment by yuhan0** [comment]
Whoops, I removed the duplicated line, still investigating the test failures (can't get them to run locally)

**Pull request merged**